rules/windows/builtin/appxdeployment_server/appxdeployment_server_mal_appx_names.yml

```
@@ -0,0 +1,24 @@
+title: Malicious AppX Package Installed
+id: 09d3b48b-be17-47f5-bf4e-94e7e75d09ce
+status: experimental
+description: Detects installation of known malicious appx packages
+references:
+    - https://www.sentinelone.com/labs/inside-malicious-windows-apps-for-malware-deployment/
+    - https://news.sophos.com/en-us/2021/11/11/bazarloader-call-me-back-attack-abuses-windows-10-apps-mechanism/
+author: Nasreddine Bencherchali
+date: 2023/01/11
+tags:
+    - attack.defense_evasion
+logsource:
+    product: windows
+    service: appxdeployment-server
+detection:
+    selection:
+        EventID: 401
+        # Add more malicious package names
+        # TODO: Investigate the packages here https://github.com/sophoslabs/IoCs/blob/master/Troj-BazarBackdoor.csv based on this report https://news.sophos.com/en-us/2021/11/11/bazarloader-call-me-back-attack-abuses-windows-10-apps-mechanism/
+        PackageFullName|contains: '3669e262-ec02-4e9d-bcb4-3d008b4afac9_96.0.1072.0_'
```

```suggestion
        PackageFullName|contains: '3669e262-ec02-4e9d-bcb4-3d008b4afac9'
```
For ref did not find the "_96.0.1072.0_" part